backend/go.mod

```diff
@@ -0,0 +1,5 @@
+module emotic
+
+go 1.14
+
+require github.com/short-d/app v0.0.0-20200627081605-eabc0539025f
```

Comment:
There are should be other indirect dependencies being added here because they are not yet using Go modules.